client/me/notification-settings/wpcom-settings/index.jsx

```diff
@@ -121,6 +130,46 @@ class WPCOMNotifications extends React.Component {
 					) }
 				/>
 
+				{ this.props.hasJetpackSites ? (
+					<>
+						<FormSectionHeading>
+							{ this.props.translate( 'Email from Jetpack' ) }
+						</FormSectionHeading>
+
+						<EmailCategory
+							name={ options.jetpack_marketing }
+							isEnabled={ get( settings, options.jetpack_marketing ) }
+							title={ translate( 'Suggestions' ) }
+							description={ translate( 'Tips for getting the most out of Jetpack.' ) }
+						/>
+
+						<EmailCategory
+							name={ options.jetpack_research }
+							isEnabled={ get( settings, options.jetpack_research ) }
+							title={ translate( 'Research' ) }
+							description={ translate(
+								'Opportunities to participate in Jetpack research and surveys.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Opportunities to participate in WordPress.com research and surveys.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).